tfe/data_source_organizations.go

```diff
@@ -53,3 +61,29 @@ func dataSourceTFEOrganizationList(d *schema.ResourceData, meta interface{}) err
 
 	return nil
 }
+
+func getAdminOrgs(client *tfe.Client) (*tfe.AdminOrganizationList, error) {
+	log.Printf("[DEBUG] Listing all organizations (admin)")
+	orgs, err := client.Admin.Organizations.List(ctx, tfe.AdminOrganizationListOptions{})
+	if err != nil {
+		if err == tfe.ErrResourceNotFound {
+			return nil, nil
```

Comment:
Returning `nil, nil` here on `ErrResourceNotFound` is intentional because of the way the code is sequenced above. We want to return an error in `dataSourceTFEOrganizationList` if there is an actual error. If we get `ErrResourceNotFound`, then we want to tell the calling function that there aren't any "admin organizations" and there isn't an error, so that way we continue on to retrieve organizations.